.github/workflows/swift_package_test.yml

```diff
@@ -110,3 +122,27 @@ jobs:
       - name: Build / Test
         timeout-minutes: 60
         run: docker run -v ${{ github.workspace }}:C:\source swiftlang/swift:${{ matrix.swift_version }}-windowsservercore-1809 cmd /s /c  "swift --version & swift test --version & cd C:\source\ & ${{ inputs.windows_build_command }} ${{ inputs.swift_nightly_flags }}"
+
+  musl-build:
+    name: MUSL
+    if: ${{ inputs.enable_musl_checks }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    container: swift:6.0-noble
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+      - name: Install SDK
+        run: swift sdk install https://download.swift.org/swift-6.0.1-release/static-sdk/swift-6.0.1-RELEASE/swift-6.0.1-RELEASE_static-linux-0.0.1.artifactbundle.tar.gz --checksum d4f46ba40e11e697387468e18987ee622908bc350310d8af54eb5e17c2ff5481
```

Comment:
do we want a hardcoded checksum/artifact url hardcoded here? Or should they be inputs